templates/annotate.sh

```diff
@@ -124,6 +127,72 @@ vep () {
   !{singularity_vep} vep "${args[@]}"
 }
 
+capice () {
```

Comment:
I suggest to split the content of this function in three functions to improve readability:
```
capice_vep
capice_bcftools
capice_predict
```
and call these functions from this capice function